integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/property/PropertyIntegrationAxis2ClientScopeRemoveProperties.java

```diff
@@ -0,0 +1,85 @@
+/*
+*Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+package org.wso2.carbon.esb.mediator.test.property;
+
+import org.apache.axiom.om.OMElement;
+import org.apache.synapse.MessageContext;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import javax.xml.namespace.QName;
+import java.io.File;
+
+public class PropertyIntegrationAxis2ClientScopeRemoveProperties extends ESBIntegrationTest{
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        super.init();
+        loadESBConfigurationFromClasspath(File.separator + "artifacts" + File.separator + "ESB" + File.separator + "synapseconfig" + File.separator + "propertyMediatorConfig" + File.separator + "scope_axis2-client_remove_config.xml");
+    }
```

Comment:
Can't we deploy the proxies at the beginning of the test module?

Reply:
There're quite a number of proxies. For better understandability, i'm loading like this. But we can follow another approach. We can have a few proxies and define multiple property mediators in each of them. Would that be a good idea?

Reply:
done